### PR TITLE
[UA update] chore: Update User-Agent string

### PR DESCRIPTION
**Pull request by 96malhar**
*Issue #, if available:*
SDK-169882

*Description of changes:*
chore: Update User-Agent string

*Testing*
Verified that the UA is `lib/aws-dotnet-messaging#0.9.0.0`

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by ashovlin — APPROVED** [review]

File: cb641346-f9f6-414d-be09-90ea0b8366a7.json
Comment (philasmar) [inline cb641346-f9f6-414d-be09-90ea0b8366a7.json:1]: This file needs to be located in the directory `.autover/changes`. If you try and create the file with the tool, it'll automatically create it in the proper directory.

File: cb641346-f9f6-414d-be09-90ea0b8366a7.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+    "Projects": [
+      {
+        "Name": "AWS.Messaging",
+        "Type": "Patch",
+        "ChangelogMessages": [
+          "chore: Update User-Agent string"
```
Comment (philasmar) [inline cb641346-f9f6-414d-be09-90ea0b8366a7.json:7]: The changelog messages in this file are supposed to be user-friendly messages, not commit messages. You shouldn't use conventional commit messages format in here.

**Review by philasmar — CHANGES_REQUESTED** [review]

File: cb641346-f9f6-414d-be09-90ea0b8366a7.json
Comment (96malhar) [reply cb641346-f9f6-414d-be09-90ea0b8366a7.json:1]: Done

File: cb641346-f9f6-414d-be09-90ea0b8366a7.json (hunk 1 above)
Comment (96malhar) [reply cb641346-f9f6-414d-be09-90ea0b8366a7.json]: Fixed

**Review by philasmar — APPROVED** [review]

**Pull request merged**